pypeit/core/wavecal/autoid.py

```diff
@@ -535,17 +524,21 @@ def run_brute(self):
         # Now that all slits have been inspected, cross match to generate a
         # master list of all lines in every slit, and refit all spectra
         if self._nslit > 1:
+            msgs.info('Checking wavelength solution by cross-correlating with all slits')
+
+            msgs.info('Cross-correlation iteration #1')
             obad_slits = self.cross_match(good_fit)
-            cntr = 0  # Introduce a counter to stop the while loop, just in case the loop gets stuck
+            cntr = 2
```

Comment:
Should this really be 2?

Reply:
The first iteration is done outside the loop.